endpoints/openrtb2/amp_auction.go

```diff
@@ -404,6 +406,17 @@ func (deps *endpointDeps) overrideWithParams(httpRequest *http.Request, req *ope
 	return nil
 }
 
+func getQueryParam(httpRequest *http.Request, name string) (string, bool) {
```

Comment:
Switched from using `httprequest.FormValue` since that also looks in the request body and can't tell the difference between a param being omitted vs set to an empty string.

Reply:
AMP requests are always GET, so body should be empty anyway. Also not set and turned off mean the same thing, so it is a distinction without a difference.

Reply:
Distinction removed.